data/json/terrain.json

```diff
@@ -849,6 +895,7 @@
         "type" : "terrain",
         "id" : "t_wall_log_broken",
         "name": "broken log wall",
+        "description": "A destroyed wood wall, with only a supporting log left, ready to collapse.  Looks repairable if some of the damage was replaced and patched together with nails and planks.",
```

Comment:
`if some of the damage was replaced` isn't grammatically correct

Reply:
It is grammatically correct, however the sentence is a bit awkward; I could just make it
>if the damage was replaced